packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.test.ts

```diff
@@ -39,46 +64,9 @@ describe("MongoDbEmbeddedContentStore", () => {
         embeddingName: OPENAI_RETRIEVAL_EMBEDDING_DEPLOYMENT,
       },
     });
-  });
-
-  afterEach(async () => {
-    await store?.drop();
-    await store?.close();
-    await mongod?.stop();
-  });
-
-  it("handles embedded content", async () => {
-    assert(store);
-
-    const page: PersistedPage = {
-      action: "created",
-      body: "foo",
-      format: "md",
-      sourceName: "source1",
-      metadata: {
-        tags: [],
-      },
-      updated: new Date(),
-      url: "/x/y/z",
-    };
-
-    const anotherPage: PersistedPage = {
-      action: "created",
-      body: "bar",
-      format: "md",
-      sourceName: "another-source",
-      metadata: {
-        tags: [],
-      },
-      updated: new Date(),
-      url: "/a/b/c",
-    };
-
-    const pages = [page, anotherPage];
     for (const page of pages) {
-      const embeddedContent = await store.loadEmbeddedContent({ page });
+      embeddedContent = await store.loadEmbeddedContent({ page });
```

Comment:
(Applies to the following line - can't comment on it directly)

We shouldn't have `expect()` calls outside of `test/it` blocks. This for loop used to be in the `it("handles embedded content", ...` block but that was removed and now it's inside of `beforeEach`.